### DISCUSSION TITLE
App Management

### DISCUSSION THREAD

**Discussion by mv-stns** [MatthiasGrandl/Loungy#39] [General]
Besides the beautiful alternative to raycast and co.
What App are you using to display all these Dots? Is this some kind of App Manager? @MatthiasGrandl 
![scrnsht_12-10 -2024at 15 25 00@2x](https://github.com/user-attachments/assets/21be44d3-c234-4581-8b6d-9ee3cff25931)

**Comment by MatthiasGrandl** [comment]
Hi! This is [Sketchybar](https://github.com/FelixKratz/SketchyBar) . The dots represent my [Aerospace workspaces](https://github.com/nikitabobko/AeroSpace). Hope this helps!
